requirements.txt

```diff
@@ -1,5 +1,40 @@
-Flask==2.2.2
-Flask-HTTPAuth==4.7.0
-requests==2.25.1
-pytest==7.2.0
-pytest-mock==3.10.0
\ No newline at end of file
+anyio==3.6.2
```

Comment:
Perhaps take another look at whether all these requirements are needed. Just because they were installed along with FastAPI doesn't necessarily mean they are used.